enhancements/oc/image-content-source-policy-awareness.md

```diff
@@ -0,0 +1,107 @@
+---
+title: image-content-source-policy-awareness
+authors:
+  - "@sallyom"
+reviewers:
+  - "@smarterclayton"
+  - "@soltysh"
+  - "@wking"
+approvers:
+  - "@smarterclayton"
+creation-date: 2020-05-19
+last-updated: 2020-07-29
+status: provisional
```

Comment:
I think with those comments addressed this is ready for implementation, if you can make the updates I'll approve this.

Reply:
I've updated w/ your feedback.